templates/featured-stories.php

```diff
@@ -21,7 +21,7 @@
 
 foreach ( $myposts as $post ) : setup_postdata( $post );
 	$child_theme_uri = get_stylesheet_directory_uri();
-	$thumbnail       = ( empty( the_post_thumbnail() ) ) ? "<img src='{$child_theme_uri}/dist/images/new-noteworthy.png' alt='new and noteworthy' />" : the_post_thumbnail( '150' );
+	$thumbnail       = ( empty( the_post_thumbnail() ) ) ? "<img src='{$child_theme_uri}/dist/images/noteworthy.jpg' alt='new and noteworthy' />" : the_post_thumbnail( '150' );
```

Comment:
if the post _does_ have a thumbnail, it needs to be given a value greater than 150. If we want to avoid relying on content authors to provide images of a certain dimension, I recommend looking at the strategy used in bcc-sage https://github.com/BCcampus/bcc-sage/blob/dev/resources/views/partials/news-feature-front.blade.php#L10